src/services/sendEmailService.ts

```diff
@@ -0,0 +1,55 @@
+const sendGridMail = require('@sendgrid/mail');
+
+import sendEmail11 from "src/controllers/sendEmailController";
+
+ var sendEmail1 = async (shifts,emailToAgent) => {
+
+sendGridMail.setApiKey('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');
+
+function getMessage() {
+  const body = 'Your next week ticket assignment hours';
```

Comment:
body does not seem to be used - please remove if its obsolete

Reply:
resolved - removed